examples/tensorflow/nlp/transformer_lt_mlperf/quantization/ptq/README.md

```diff
@@ -131,51 +130,52 @@ class Dataset(object):
 
-### Write Yaml config file
-In examples directory, there is a transformer_lt_mlperf.yaml for tuning the model on Intel CPUs. The 'framework' in the yaml is set to 'tensorflow'. If running this example on Intel GPUs, the 'framework' should be set to 'tensorflow_itex' and the device in yaml file should be set to 'gpu'. The transformer_lt_mlperf_itex.yaml is prepared for the GPU case. We could remove most of items and only keep mandatory item for tuning. We also implement a calibration dataloader and have evaluation field for creation of evaluation function at internal neural_compressor.
-
-```yaml
-model:
-  name: transformer_lt_mlperf
-  framework: tensorflow
-  inputs: input_tokens
-  outputs: model/Transformer/strided_slice_15
-
-device: cpu                                          # optional. default value is cpu, other value is gpu.
-
-quantization:
-  calibration:
-    sampling_size: 500
-  model_wise:
-    weight:
-      granularity: per_channel
-
-tuning:
-  accuracy_criterion:
-    relative: 0.01
-  exit_policy:
-    timeout: 0
-    max_trials: 100
-  random_seed: 9527
+### Quantization Config
+The Quantization Config class has default parameters setting for running on Intel CPUs. If running this example on Intel GPUs, the 'backend' parameter should be set to 'itex' and the 'device' parameter should be set to 'gpu'.
+
+```
+config = PostTrainingQuantConfig(
+    device="gpu",
+    backend="itex",
+    inputs=['input_tokens'],
+    outputs=['model/Transformer/strided_slice_15'],
+    ...
+    )
 ```
 
-Here we set the input tensor and output tensors name into *inputs* and *outputs* field.
+Here we set the input tensor and output tensors name into *inputs* and *outputs* args.
 In this case we calibrate and quantize the model, and use our calibration dataloader initialized from a 'Dataset' object.
 
 ### Code update
 After prepare step is done, we add tune code to generate quantized model.
 
+#### Tune
 ```python
-from neural_compressor.experimental import Quantization, common
-quantizer = Quantization(FLAGS.config)
-dataset = Dataset(FLAGS.input_file, FLAGS.vocab_file)
-quantizer.calib_dataloader = common.DataLoader(dataset,
-                                               collate_fn = collate_fn,
-                                               batch_size = FLAGS.batch_size)
-quantizer.model = common.Model(graph)
-quantizer.eval_func = eval_func
-q_model = quantizer.fit()
-q_model.save(FLAGS.output_model)
+  from neural_compressor import quantization
+  from neural_compressor.experimental import common
+  from neural_compressor.config import PostTrainingQuantConfig
+  dataset = Dataset(FLAGS.input_file, FLAGS.vocab_file)
+  calib_dataloader = common.DataLoader(dataset,
+                                        collate_fn = collate_fn,
+                                        batch_size = FLAGS.batch_size)
+  input_model = common.Model(graph)										
```

Comment:
is it a pb? if yes, you can directly pass graph into fit() but not wrap with common.Model

Reply:
It's a graph_def. The common.Model has been removed. Thanks for comments.